lang/pl/index.md

```diff
@@ -244,22 +310,49 @@ w semantycznym wersjonowaniu chodzi przede wszystkim o przekazanie znaczenia
 zmiany poprzez zmianę numeru wersji. Jeśli zmiany są ważne dla użytkowników,
 poinformuj ich o tym poprzez numer wersji.
 
-### Jak powinienem radzić sobie z dezaprobowaniem funkcjonalności?
+### Jak powinienem radzić sobie z wycofywaniem funkcjonalności?
 
-Dezaprobowanie istniejącej funkcjonalności jest normalną częścią programowania
+Wycofywanie istniejącej funkcjonalności jest normalną częścią programowania
 i często jest konieczne, by móc rozwijać oprogramowanie. Gdy wycofujesz część
 swojego publicznego API, powinieneś zrobić dwie rzeczy: (1) zaktualizować
 dokumentację, by użytkownicy wiedzieli o tej zmianie, (2) wypuścić nowe wydanie
-minor z informacją o zdezaprobowaniu. Zanim całkowicie usuniesz funkcjonalność
+minor z informacją o wycofaniu. Zanim całkowicie usuniesz funkcjonalność
 w nowym wydaniu major, powinno być co najmniej jedno wydanie minor zawierające
-zdezaprobowanie, aby użytkownicy mogli płynnie przejść na nowe API.
+informację o wycofaniu, aby użytkownicy mogli płynnie przejść na nowe API.
 
-### Czy semver ma limit długości na oznaczenie wersji?
+### Czy SemVer ma limit długości na oznaczenie wersji?
 
 Nie, ale miej zdrowy rozsądek. Na przykład numer wersji długi na 255 znaków to
 prawdopodobnie przesada. Ponadto konkretne systemy mogą narzucać swoje własne
 ograniczenia na rozmiar tego ciągu znaków.
 
+### Czy ciąg „v1.2.3” spełnia zasady?
+
+Nie, „v1.2.3” nie jest zgodne z wersjonowaniem semantycznym. Jednak przedrostek „v” jest
+zazwyczaj używany w języku angielskim do oznaczenia, że mamy do czynienia z numerem wersji.
+Skrót „v” od angielskiego słowa „version” jest często spotykany w systemach kontroli wersji.
+Przykładowo, w przypadku `git tag v1.2.3 -m "Release version 1.2.3"`, „v1.2.3” jest nazwą taga,
+a semantyczna wersja to „1.2.3”.
+
+### Czy istnieje sugerowane wyrażenie regularne (RegEx) do weryfikacji porawności SemVer?
+
+Istnieją dwa. Pierwsze wykorzystuje grupy nazwane i jest przeznaczone dla systemów, które
+wspierają tę funkcjonalność (PCRE [Perl Compatible Regular Expressions, np. Perl, PHP i R], Python czy Go).
```

Comment:
(nit)
```suggestion
wspierają tę funkcjonalność (PCRE [Perl Compatible Regular Expressions, np. Perl, PHP i R], Python i Go).
```

Reply:
The linguistic problem here is that when we us `i` then we create a closed set of languages. In original text 

> (PCRE [Perl Compatible Regular Expressions, i.e. Perl, PHP and R], Python and Go)

we have `and`, that could be translated as `i` (and the most of a people do that this way), but context is that we would like to present a subset of languages that support mentioned functionality. So we can use word `lub` but when we make translation PL»EN then we change original text. However, when we use `czy` then translation PL»EN will give us original text. 
In this context, Polish `czy` is not English `whether` (direct translation). 
In this [article](https://sjp.pwn.pl/poradnia/haslo/czy-jako-spojnik;14088.html) you have mentioned that `czy` could be used as equivalent of `i` or `lub`, depend on context.